src/plugins/exec/sim-host-handlers.js

```diff
@@ -87,3 +96,19 @@ function handleSavedSim(success, fail, service, action) {
     }
     return false;
 }
+
+function sendExecUnhandledTelemetry(service, action, hasPersisted, isSuccess, hasResult, saveResult) {
+    var props = {
+        'service': service,
+        'action': action,
+        'has-persisted-result': hasPersisted,
+        'is-success': isSuccess,
+        'has-result-value': hasResult
+    };
+
+    if (typeof saveResult !== 'undefined') {
+        props['save-result'] = saveResult;
```

Comment:
I feel like in this case, the absence of `save-result` is not the same as `save-result = false`. The reason is that if `has-persisted-value` is `true`, then `save-result` should be ignored because the user wasn't even presented the choice of whether to persist the call or not. The dialog box didn't even pop up, because that call result was already persisted.

So, I was just afraid that default this to `false` might be misleading when querying telemetry data. For example, searching on "WHERE save-result = FALSE" might give the impression that 75% of the users chose not to persist their unhandled exec call, when in reality half of those weren't even presented the choice.

I hope I'm not being too confusing :) I'll add JSDoc headers to explain the parameters, which should help!